backon/src/backoff/constant.rs

```diff
@@ -90,6 +98,10 @@ impl BackoffBuilder for ConstantBuilder {
 
             attempts: 0,
             jitter: self.jitter,
+            #[cfg(not(feature = "std"))]
```

Comment:
Hi, sorry for commenting on this again. The behaivor here is not as expected:

```rust
if let Some(seed) = self.seed {
  fastrand::Rng::with_seed(seed)
} else { 
   #[cfg(feature = "std")]
   {
     fastrand::Rng::new()
   }
   #[cfg(not(feature = "std"))]
   {
     fastrand::Rng::with_seed(super::RANDOM_SEED)
   }
}
```

Reply:
Yes, that is what you wrote. Sorry for the inaccuracy.